src/index.js

```diff
@@ -1,12 +1,4 @@
-import CSSTransition from './CSSTransition';
-import ReplaceTransition from './ReplaceTransition';
-import TransitionGroup from './TransitionGroup';
-import Transition from './Transition';
-
-
-module.exports = {
-  Transition,
-  TransitionGroup,
-  ReplaceTransition,
-  CSSTransition,
-};
+export { default as CSSTransition } from './CSSTransition';
```

Comment:
This changes the semantics for existing users, which would be a breaking change.

Reply:
How does that change the result of below?:
```js
import { Transition, CSSTransition, TransitionGroup } from 'react-transition-group';
// or cjs
const { Transition, CSSTransition, TransitionGroup } = require('react-transition-group');
```
Transpiled code works exactly the same as before.